pandas/io/formats/describe.py

```diff
@@ -0,0 +1,464 @@
+"""Module responsible for execution of NDFrame.describe() method.
+
+Method NDFrame.describe() delegates actual execution to function describe_ndframe().
+
+Strategy pattern is utilized.
+ - The appropriate strategy is selected based on the series datatype.
+ - The strategy is responsible for running proper description.
+"""
+
+from abc import ABC, abstractmethod
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    Union,
+    cast,
+)
+import warnings
+
+import numpy as np
+
+from pandas._libs.tslibs import Timestamp
+from pandas._typing import Dtype, FrameOrSeries, FrameOrSeriesUnion, Label
+from pandas.util._validators import validate_percentile
+
+from pandas.core.dtypes.common import (
+    is_bool_dtype,
+    is_datetime64_any_dtype,
+    is_numeric_dtype,
+    is_timedelta64_dtype,
+)
+
+import pandas as pd
+
+from pandas.io.formats.format import format_percentiles
+
+if TYPE_CHECKING:
+    from pandas import DataFrame, Series
+
+
+def describe_ndframe(
+    *,
```

Comment:
why is the * needed here?

Reply:
Just to enforce keyword-only arguments, to avoid any confusion with positional arguments.